### PR TITLE
fix new gosec issues

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes https://github.com/IBM-Cloud/power-go-client/issues/215

Signed-off-by: Yussuf Shaikh <yussuf.shaikh1@ibm.com>

### CODE REVIEW

**Review by powervs-ibm — APPROVED** [review]
Looks good!

File: examples/instance/main.go [hunk 1]
```diff
@@ -74,15 +74,20 @@ func main() {
...
+	if len(*createRespOk) == 0 {
+		log.Fatal("create response is empty")
+	}
 	log.Printf("***************[1]****************** %+v\n", *createRespOk)
 
-	insIDs := make([]string, 0)
+	insID := ""
 	for _, in := range *createRespOk {
-		insID := in.PvmInstanceID
-		insIDs = append(insIDs, *insID)
```
Comment (ismirlia) [inline examples/instance/main.go:82]: We don't need to append the insID in the new version?

Comment (yussufsh) [reply examples/instance/main.go:82]: This example only creates 1 instance hence we can directly use the first instance response from the response list.

Comment (ismirlia) [reply examples/instance/main.go:82]: Ah okay looks good!

**Review by ismirlia — APPROVED** [review]

**Pull request merged**